.github/workflows/build_test.yml

```diff
@@ -120,12 +120,22 @@ jobs:
     runs-on: ubuntu-latest
     if: github.event_name == 'pull_request'
     steps:
+      - name: Checkout cycamore
+        uses: actions/checkout@v4
+
       - name: Save PR number to file
         run: |
           echo "${{ github.event.number }}" > pr_number
+          echo "${{ github.event.pull_request.head.sha }} - $(git log -1 --format=%ci)" > commit_hash_timestamp
       
-      - name: Upload artifact
+      - name: Upload PR number artifact
         uses: actions/upload-artifact@v4
         with:
           name: pr_number
-          path: pr_number
\ No newline at end of file
+          path: pr_number
+
+      - name: Upload Commit hash artifact
+        uses: actions/upload-artifact@v4
+        with:
+          name: commit_hash

```

Comment:
doesn't this need to be `commit_hash_timestamp` now?

Reply:
Yes! Thanks for catching that